### ISSUE TITLE
API responds with outdated information

### ISSUE THREAD

**Issue by andrew-demb** [asgrim/year#42]
Now is 2024 in my calendar, but "en/currentYear" responds with 2023

**Comment by asgrim** [comment]
I agree. This is problematic for everyone. I think the fix is very complex though, it might take a few months to resolve

**Comment by asgrim** [comment]
Thankfully, this longstanding issue is finally fixed. Thanks to the epic hard work of multiple teams, across multiple countries, for many months.
